### ISSUE TITLE
Work with web pages also?

### ISSUE THREAD

**Issue by antomonte** [elias-sundqvist/obsidian-annotator#59]
Hi! The plugin is wonderful, but I think it could be hugely improved just adding the ability of working also with webpages. 

So, the webpage could be rendered (maybe using iframes? don't know...) and we just go along highlighting, taking notes... and all of it **automatically**  stored on markdown within our vault (as currently it does for PDF files)

Thanks!

**Comment by GitMurf** [comment]
Oh wow this would be cool!! A Web Highlighter as well! Now that would definitely get some extra attention!

**Comment by joellti** [comment]
Can we support saved html files to start with? I think CORS issue doesn't affect locally saved html files.

**Comment by elias-sundqvist** [comment]
> Can we support saved html files to start with? I think CORS issue doesn't affect locally saved html files.

I implemented a CORS workaround for computers in release 1.3, so that is no longer an issue. I just need to find the time to work on the plugin. 

Here is a proof-of-concept screenshot. Most websites are a bit glitchy, but it kind of works. 
![image](https://user-images.githubusercontent.com/9102856/135342050-6b918f72-9678-40e2-855b-892af88c1f3c.png)

**Comment by antomonte** [comment]
Really great! keep it going!

**Comment by antomonte** [comment]
Hi!

is there any update on this topic?

**Comment by elias-sundqvist** [comment]
> 
> 
> Hi!
> 
> is there any update on this topic?

Hi, I'm sorry to say that I haven't found much time to work on this plugin these past months. 
I did most of the development of this plugin during my summer break, and when my studies at the university started again things have stagnated significantly. 
The web page annotations are still not really at a useable state. 

If I'm not too occupied this weekend, I will push my current code and add [BRAT](https://github.com/TfTHacker/obsidian42-brat) so that these newer/less stable features can be tested. 

I'm not exactly sure when I will get much free time to work on the plugin again, but rest assured that I haven't abandoned it.

**Comment by antomonte** [comment]
Sure!

you have your own priorities, so do not feel bad about it.

I think that for us it is enough just knowing that the project will not fall forgotten, and from time to time you'll spend some available time... and maybe in the future we can count with one of the **best** plugins we'll ever see.

Best regards

**Comment by Seraphli** [comment]
Hi @elias-sundqvist, any update about this feature?

**Comment by elias-sundqvist** [comment]
Yes, I should be able to make a beta release with this feature in one or two weeks.  All websites don't work, but many do.

**Comment by elias-sundqvist** [comment]
Beta 0.2.1 is now released with some basic web annotation support. 
A major limitation right now is that you cannot navigate by clicking on links. 
Also, this feature does not work on mobile at the moment. But it should start working as soon as the latest API features become available for mobile.

**Comment by prakersh** [comment]
If anyone here is specifically looking for web page annotation, [onUI](https://github.com/onllm-dev/onUI) is a free, open-source browser extension (Chrome, Edge, Firefox) built exactly for that — you can annotate any live webpage with text notes or freehand drawing directly in the browser. It won't store to your Obsidian vault, but it does the web annotation part well.

**Comment by huyz** [comment]
@prakersh Since this has nothing to do with Obsidian, your comment is just spam
